views/editExistingSightingForm.ejs

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html>
+  <title>UFO Sighting</title>
+  <link
+    rel="stylesheet"
+    href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+    integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2"
+    crossorigin="anonymous"
+  />
+  <link rel="stylesheet" href="/styles.css" />
+
+  <!--NavBar-->
+  <nav class="navbar navbar-expand-sm fixed-top navbar-light navBarBg">
+    <a class="navbar-brand">UFO Sightings!</a>
+    
+  <!-- Toggler/collapsibe Button -->
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsibleNavbar">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+ <!-- Navbar links -->
+    <div class = "collapse navbar-collapse" id="collapsibleNavbar">
+      <ul class="navbar-nav">
+        <li class="nav-item active">
+          <a class="nav-link" href="/">Home</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="/shapes">Shapes</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="/newSighting">Report New Sighting</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
+
+  <!-- Body-->
+  <body>
+    <div class="container navTopMargin">
+      <div class="row">
+        <div class="col d-flex justify-content-center submitSighting">
+          Edit this sighting
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-12 d-flex justify-content-center">
+          <form action="/sighting/<%=id%>/edit?_method=PUT" method="POST">
+            <div class="form-group required">
+              <label class= "control-label" for="description">Description:</label>
+               <!-- Request.body object will reflect a singular sighting object, with the keys that will be transformed into local 
+                variables e.g (description, date_time,shape etc) -->
+              <!--Feedback is a new local variable (obj) which reflects the server-side feedback if the form is valid or not -->
+              <!-- Hence, to check if fields are valid, we check if "feedback" local variable (obj) exists in the request.body object i.e "locals"-->
+              <!-- If present, if description value is not "ok" select is-invalid state -->
+              <input
+                type="text"
+                class="form-control
+              <%if(locals.feedback){%>
```

Comment:
Why is it necessary to check if `locals.feedback` exists and not just `feedback`?

Reply:
because if I just type in `<% feedback %>` , what this gives me is the value inside the local variable `feedback`. What I wanted to check is the presence of the local variable `feedback`.